src/users/list_users/v2.rs

```diff
@@ -116,4 +123,8 @@ pub struct UserMinorDetails {
     /// The user's avatar URL, if set.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub avatar_url: Option<String>,
+
+    /// Whether the account is locked.
+    #[serde(deserialize_with = "crate::serde::bool_or_uint")]
+    pub locked: bool,
```

Comment:
This also needs `serde(default)`, right? (so responses from older synapses can be deserialized)

Reply:
right :( sry I missed it!

Reply:
No problem! I can actually fix it in the web UI..